assets/sass/okta-theme.scss

```diff
@@ -52,19 +52,6 @@
     text-decoration: none;
     cursor: pointer;
   }
-
```

Comment:
I am guessing this has been here for a while ? since we release 3.0 ?

Reply:
this was added in the OIE commit.